node/node.go

```diff
@@ -109,10 +114,11 @@ type Node struct {
 	Syncer       nodeChainSyncer
 	PowerTable   consensus.PowerTableView
 
-	BlockMiningAPI *block.MiningAPI
-	PorcelainAPI   *porcelain.API
-	RetrievalAPI   *retrieval.API
-	StorageAPI     *storage.API
+	BlockMiningAPI      *block.MiningAPI
+	PorcelainAPI        *porcelain.API
+	RetrievalAPI        *retrieval.API
+	StorageAPI          *storage.API
+	StorageFaultSlasher storageFaultSlasher
```

Comment:
Nit: I would put this after some other "worker" implementations, like the mining worker or storage/retrieval miners.